src/main/resources/explanations/challenge36_hint.adoc

```diff
@@ -0,0 +1,6 @@
+This challenge can be solved by retrieving the strings in
+
+1. Follow the following steps:
+- Extract all the strings of the executable file
```

Comment:
Can you provide a description for the methods for this using R2 or Ghidra for instance please?

Reply:
Right